python/config/bradley.py

```diff
@@ -0,0 +1,45 @@
+def match_containers(alma_items: list, aspace_containers: list, logger) -> tuple:
+    unmatched_alma_items = []
+    matched_aspace_containers = []
+    for item in alma_items:
+        item_id = item.get("item_data").get("pid")
+        barcode = item.get("item_data").get("barcode")
+        description = item.get("item_data").get("description")
+        alma_container_type = description.split(".")[0]
+        alma_indicator = description.split(".")[1]
+        # if indicator starts with 0, remove it
+        while alma_indicator[0] == "0":
+            alma_indicator = alma_indicator[1:]
+        # if indicator ends with " RESTRICTED", remove it
+        if alma_indicator.endswith(" RESTRICTED"):
+            alma_indicator = alma_indicator[:-11]
+        # match with ASpace top container based on container type and indicator
+        for tc in aspace_containers:
+            tc_type = tc.get("type")
+            tc_indicator = tc.get("indicator")
+            tc_uri = tc.get("uri")
+            if tc_type == alma_container_type and tc_indicator == alma_indicator:
```

Comment:
Once there are 2 dictionaries of tuples for matching, this comparison can be more generic:
```
for key in aspace_keys:
    if key in alma_keys:
       # do something
```